leftwm-core/src/handlers/window_handler.rs

```diff
@@ -17,22 +19,22 @@ impl<C: Config, SERVER: DisplayServer> Manager<C, SERVER> {
         }
 
         let mut is_first = false;
+        let mut on_same_tag = false;
         //Random value
         let mut layout: Layout = Layout::MainAndVertStack;
-        let is_scratchpad = is_scratchpad(self, &window);
         setup_window(
             self,
             &mut window,
-            x,
-            y,
-            is_scratchpad,
+            (x, y),
             &mut layout,
             &mut is_first,
+            &mut on_same_tag,
         );
-        insert_window(self, &mut window, is_scratchpad, layout);
+        insert_window(self, &mut window, layout);
 
         let follow_mouse = self.state.focus_manager.focus_new_windows
-            || self.state.focus_manager.behaviour == FocusBehaviour::Sloppy;
+            && self.state.focus_manager.behaviour == FocusBehaviour::Sloppy
+            && on_same_tag;
```

Comment:
I don't get it right now, why you changed this OR to AND.

Reply:
Cause if focus_new_windows was false, it would still move the mouse to the window (in turn focusing it) if in focus_behaviour="Sloppy".

Reply:
So actually also a passing by bug fix, right?